modules/ROOT/pages/indexes/search-performance-indexes/using-indexes.adoc

```diff
@@ -243,6 +243,28 @@ Text indexes are only used for exact query matches. To perform approximate match
 
 For more information about the predicates supported by text indexes, see xref:indexes/search-performance-indexes/managing-indexes.adoc#text-indexes-supported-predicates[Create, show, and delete indexes -> Text indexes: supported predicates].
 
+[[text-index-dictionary-variables]]
+=== Text indexes and dictionary variables
```

Comment:
"Dictionary" and "map" are often used interchangeable, in Cypher we call them "maps" - https://neo4j.com/docs/cypher-manual/current/values-and-types/maps/